src/worker/core/team/valueChange.ts

```diff
@@ -588,4 +624,53 @@ const valueChange = async (
 	return valuesAdd - valuesRemove;
 };
 
+const getModifiedPickRank = async (
+	tid: number,
+	pidsAdd: number[],
+	pidsRemove: number[],
+) => {
+	const teamSeason = await idb.cache.teamSeasons.indexGet(
+		"teamSeasonsBySeasonTid",
+		[tid, g.get("season")],
+	);
+	const teams = g.get("numActiveTeams");
+	const gp = teamSeason?.gp ?? 0;
+	const seasonFraction = gp / g.get("numGames");
+
+	const players = await idb.cache.players.indexGetAll("playersByTid", tid);
+	const playersAfterTrade = players.filter(p => !pidsRemove.includes(p.pid));
+	for (const pid of pidsAdd) {
+		const p = await idb.cache.players.get(pid);
+		if (p) {
+			playersAfterTrade.push(p);
+		}
+	}
+	const playerRatings = playersAfterTrade.map(p => ({
+		pid: p.pid,
+		value: p.value,
+		ratings: {
+			ovr: p.ratings.at(-1)!.ovr,
+			ovrs: p.ratings.at(-1)!.ovrs,
+			pos: p.ratings.at(-1)!.pos,
+		},
+	}));
+
+	const newTeamOvr = team.ovr(playerRatings);
+	const newTeamOvrRank =
+		newTeamOvr < cache.teamOvrs[cache.teamOvrs.length - 1].ovr
+			? cache.teamOvrs.length - 1
+			: cache.teamOvrs.findIndex(t => t.ovr < newTeamOvr);
+	const newTeamOvrWinp = 0.25 + (0.5 * (teams - 1 - newTeamOvrRank)) / teams;
```

Comment:
Move this formula to a function so it's not repeated twice, otherwise they risk falling out of sync in the future.